package.json

```diff
@@ -68,6 +68,9 @@
       "internal": ":house: Internal",
       "documentation": ":memo: Documentation"
     },
-    "ignoreCommitters": ["renovate-bot", "dependabot"]
+    "ignoreCommitters": [
```

Comment:
was this changed accidentally?

Reply:
I will revert it back. My IDE is doing some weird things